app/views/wiki/edit.html.erb

```diff
@@ -21,6 +21,13 @@
       <li><a href="/wiki/authoring-help#Advanced+formatting"><%= t('wiki.edit.advanced_formatting') %></a></li>
     </ul>
   </div>
+  <hr />
+  <div class="sidebar-nav hidden-sm">
```

Comment:
Nice work with this, just a few changes and it's ready for merge.Thanks @Ronaq13 .